libfreerdp/core/gateway/rts.c

```diff
@@ -887,7 +887,7 @@ int rts_recv_flow_control_ack_pdu(rdpRpc* rpc, BYTE* buffer, UINT32 length)
 	BYTE ChannelCookie[16];
 
 	offset = 24;
-	offset += rts_flow_control_ack_command_read(rpc, &buffer[offset], length - offset,
```

Comment:
I think the comment would be a good compromise between readability (which is the purpose of updating a variable that is not used after) and clang warning (which doesn't understand readability concerns ;) )